ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/util/SqlExecutor.scala

```diff
@@ -77,13 +96,8 @@ final class SqlExecutor(noOfThread: Int, loggerFactory: NamedLoggerFactory, metr
   }
 
   override def close(): Unit = executor.shutdown()
-
 }
 
 object SqlExecutor {
-  def apply(
-      noOfThread: Int,
-      loggerFactory: NamedLoggerFactory,
-      metrics: MetricRegistry): SqlExecutor =
-    new SqlExecutor(noOfThread, loggerFactory, metrics)
+  val maxTransientFailureCount: Int = 3
```

Comment:
Are we following the Scala style guide? In case we are this should be `MaxTransientFailureCount`:
https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods

Reply:
Yup, you're right.